presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -2341,6 +2341,12 @@ public void dropRole(ConnectorSession session, String role)
         return accessControlMetadata.listRoles(session);
```

Comment:
@electrum Can you please take a look at this pull request too? I would like to have a second pair of eyes before adding any new table to `information_schema`?